recipes-apps/xcsoar/xcsoar-testing_git.bb

```diff
@@ -52,6 +52,7 @@ SRC_URI = " \
 	file://0001_no_version_lua.patch \
 	file://0001-avoid-tail-cut.patch \
 	file://0007-Disable-touch-screen-auto-detection.patch \
+	file://0001-FreeVario \

```

Comment:
I wonder if it actually builds. There is no file named `0001-FreeVario`. There is one named `0001-FreeVario.patch`. Did you try to build it?

Reply:
Sorry, wrong file name! I correct it now!